docoptTests/DocoptTestCaseParser.swift

```diff
@@ -24,10 +24,15 @@ public struct DocoptTestCaseParser {
     }
     
     private func removeComments(string: String) -> String {
-        let removeCommentsRegEx: NSRegularExpression? = NSRegularExpression(pattern: "(?m)#.*$", options: .allZeros, error: nil)
+        let removeCommentsRegEx: NSRegularExpression?
```

Comment:
actually, we know that this particular constructor can never return nil. i've fixed this to use try! instead of try and removed the entire conditional block.